package.json

```diff
@@ -58,7 +58,7 @@
   },
   "keywords": [],
   "dependencies": {
-    "amqplib": "~0.5.1",
+    "amqplib": "~0.10.3",
```

Comment:
This should be also mentioned in the CHANGES_NEXT_RELEASE file. Something like this:

```
- Upgrade amqplib dependency from ~0.5.1 to ~0.10.3
```